helm_deploy/values-preprod.yaml

```diff
@@ -40,6 +40,8 @@ env:
   PATHFINDER_UI_URL: https://preprod.pathfinder.service.justice.gov.uk/
   PATHFINDER_ENDPOINT_API_URL: https://preprod-api.pathfinder.service.justice.gov.uk/
   REDIS_ENABLED: true
+  SOC_URL: https://manage-soc-cases-preprodservice.justice.gov.uk
+  SOC_API_ENABLED: false
```

Comment:
No dot between preprod and service. Should be manages-soc-cases-preprod.service.justice.gov.uk for now. Though it will probably change domain name to reside under hmpps.service.justice.gov.uk soon.

Reply:
Sorted :)